css-ecommerce-page/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link rel="stylesheet" href="styles.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <h1>New Arrivals</h1>
+    <div class="row">
+      <div class="column-one-fifth">
+        <img src="images/dress1.jpeg">
+      </div>
+      <div class="column-one-fifth">
+        <img src="images/ysl-bag.jpeg">
+      </div>
+      <div class="column-one-fifth">
+        <img src="images/key.jpeg">
+      </div>
+      <div class="column-one-fifth">
+        <img src="images/lv-bag.jpg">
+      </div>
+      <div class="column-one-fifth">
+        <img src="images/dress2.jpeg">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-fifth">
+        <span class="bold-text">Free People</span>
+        <span class="font-weight-500 gray-text border-left padding-left">Size 6</span>
+        <div class="price">
```

Comment:
`div` is not a text element, you should use something like `p`. If you are doing this because of the margin on `p` elements, you can always reduce or remove the margin using CSS